src/darwin/Framework/CHIP/MTRSetupPayload.h

```diff
@@ -27,6 +27,7 @@ typedef NS_OPTIONS(NSUInteger, MTRDiscoveryCapabilities) {
     MTRDiscoveryCapabilitiesSoftAP = 1 << 0, // Device supports WiFi softAP
     MTRDiscoveryCapabilitiesBLE = 1 << 1, // Device supports BLE
     MTRDiscoveryCapabilitiesOnNetwork = 1 << 2, // Device supports On Network setup
+    MTRDiscoveryCapabilitiesNFC = 1 << 3, // Device supports NFC-based commissioning
```

Comment:
```suggestion
    MTRDiscoveryCapabilitiesNFC MTR_NEWLY_AVAILABLE = 1 << 4, // Device supports NFC-based commissioning
```
so when we add PAF in here things line up properly.  But note also the availability annotation.

Reply:
I'm going to commit your suggestion but I'm afraid I don't understand it.